doc/releases.md

```diff
@@ -0,0 +1,51 @@
+
+## badge-maker package
+
+We follow [Semantic Versioning](https://semver.org/) for the badge-maker package, and publish git Tags using the form `X.Y.Z`, and all such tags of this form are badge-maker releases (e.g. [3.3.0](https://github.com/badges/shields/releases/tag/3.3.0)
+
+The [badge-maker][npm package] is also published to the npm.js registry.
+
+Releases of badge-maker are done as and when needed, and not on any predetermined interval.
+
+## Shields.io service
+
+The [Shields.io Service][shields.io] consists of the frontend [https://shields.io][shields.io] website which allows users to browse and discover available badges, as well as the badge server backend that serves up requested badges (e.g. https://img.shields.io/badge/badges-rock-blue).
+
+This is the core, free, anonymous service available for anyone to use and which serves up more than 500 million badges every month!
+
+We do not have a fixed schedule for deploying updates to the Shields.io production environment, but we typically deploy the latest version at least once per week.
+
+More information about the production environment can be found [here][production hosting]
+
+## Shields server
+
+Some users may wish to host their own instance of Shields so we also make it possible for users to do so. This is particularly useful if you want to serve badges for resources that require authentication or are not exposed to the internet (e.g: inside a corporate network). A variety of options are available either by installing from source or using a docker image.
+
+### Important information
+
+This Shields server is the exact same codebase that powers the main Shields.io service; we do not have a separate self-hosted version of Shields. This means that the server codebase is geared towards the development, maintenance, and operation of the Shields.io service so there are a few things to note:
+
+- We often have to reject or de-prioritize feature requests that are only applicable for self-hosting and/or which may be problematic for the core Shields.io offering (e.g. requiring auth on the badge server)
+- We do not accept new badges that cannot be utilized with Shields.io (e.g. those that would always require user-specific authorization)
+- We do not do any additional testing nor validation of self-hosted instances (e.g. we test Shields.io with Jira Cloud badges, but we don't test a self-hosted Shields server against a self-hosted Jira server instance with auth)
+- We're happy to try to offer some tips and guidance to self-hosters when and where we can, but we don't have the ability to troubleshoot/debug/support/etc. self-hosted Shields servers
+  - Note that we may offer some sort of paid support arrangement in the future for those that would be interested
```

Comment:
On this last point "paid support arrangement".. I know one maintainer has done this, but this is definitely not a service I offer and I wouldn't want to be contacted by someone asking for it. Dunno where you stand on it.

I think if we're going to formalise this its important to specify who to contact about this/on what basis/etc rather than this which is a bit woolly. If we're not in a position to do that (yet), lets drop the mention of it, at least for now.

Reply:
> 

On this last point "paid support arrangement".. I know one maintainer has done this, but this is definitely not a service I offer and I wouldn't want to be contacted by someone asking for it. Dunno where you stand on it.

I think if we're going to formalise this its important to specify who to contact about this/on what basis/etc rather than this which is a bit woolly. If we're not in a position to do that (yet), lets drop the mention of it, at least for now.

Good point, I'm definitely in the same boat myself as far as my willingness/capacity for this. @paulmelnikow has brought this possibility up a few times (including a few more times of late) and then of course it actually happened so it seemed like it was picking up some steam.

Paul if you're in a position where you'd like to offer this paid support stuff then I'm happy to try to frame the wording here (provided you can share whatever contact channel), but if this is still an unknown or a possible-but-not-yet ready then I'd agree with Chris and will remove

Reply:
I've done this once, and am willing to do it again. I don't have a web page up or any streamlined way to pay for a support request, so for the moment I think it is okay for folks who want this to email me at paul@m6ize.com. I'm happy to include that here if it makes sense to other folks.

Reply:
Tweaked this text to point them specifically to you Paul. Used the same pattern we added to the CoC with a badge for the email address, but happy to change this to standard link/mailto if you'd prefer